src/style.css

```
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

```suggestion
}

.stars__star:last-child {
  margin-right: 0;
}
```

reset margin for last child